web_src/js/features/copycontent.js

```diff
@@ -0,0 +1,59 @@
+import {copyToClipboard} from './clipboard.js';
+import {showTemporaryTooltip} from '../modules/tippy.js';
+import {convertImage} from '../utils.js';
+const {i18n} = window.config;
+
+async function doCopy(content, btn) {
+  const success = await copyToClipboard(content);
+  showTemporaryTooltip(btn, success ? i18n.copy_success : i18n.copy_error);
+}
+
+export function initCopyContent() {
+  const btn = document.getElementById('copy-content');
+  if (!btn || btn.classList.contains('disabled')) return;
```

Comment:
Isn't that the wrong condition?
Shouldn't that be
```suggestion
  if (!btn || btn.classList.contains('disabled') || (btn.getAttribute("disabled") && btn.getAttribute("disabled") !== "")) return;
```
?

Reply:
It's not a button but an `<a>` which does not support the `disabled` attribute, so we are using class as an alternative.